app/team.go

```diff
@@ -253,6 +253,7 @@ func (a *App) sendTeamEvent(team *model.Team, event string) {
 	sanitizedTeam.Sanitize()
 
 	message := model.NewWebSocketEvent(event, "", "", "", nil)
+	message.Broadcast.ContainsSensitiveData = true
```

Comment:
This prevents the event from being sent to anyone other than system admins. Is that reasonable? How does the webapp behave for non-admins when a team is edited?

Reply:
@jupenur crap. I think you are right. Might have to rethink this

Reply:
Good point. Also, this affects UPDATE_TEAM, DELETE_TEAM and RESTORE_TEAM events

Reply:
@jupenur done. Please review